Formula/n/nginx.rb

```diff
@@ -146,11 +146,7 @@ def caveats
   end
 
   service do
-    if OS.linux?
-      run [opt_bin/"nginx", "-g", "'daemon off;'"]
```

Comment:
I think this was needed for the generated service file too, which Homebrew/brew#15875 doesn't handle.

Reply:
Well spotted, have opened https://github.com/Homebrew/brew/pull/15880 to handle.